data/bedrock/latest/proto.yml

```diff
@@ -966,7 +966,7 @@ packet_entity_event:
       71: balloon_pop
       72: treasure_hunt
       73: agent_summon
-      74: charged_crossbow
+      74: charged_item
```

Comment:
This doesn't seem related to a feature change, it seems like a rename for clarity. Let's either revert this or apply it to all the old versions too for consistency.